app/src/main/java/com/example/cs4520_twitter/LoginScreen.kt

```diff
@@ -0,0 +1,172 @@
+                fontSize = 30.sp,
+                style = TextStyle(shadow = Shadow(blurRadius = 3f)),
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier
+                    .constrainAs(babbleTitle) {
+                        top.linkTo(parent.top, margin = (maxHeight/15).dp)
+                        absoluteLeft.linkTo(
+                            parent.absoluteLeft,
+                            margin = (maxWidth/2.6).dp
+                        )
+                    })
+
+            // user desc.
+            TextField(
+                value = usernameText,
+                onValueChange = { usernameText = it },
+                singleLine = true,
+                colors = TextFieldDefaults.colors(
+                    Color.Black,
+                    focusedContainerColor = Color.White,
+                    unfocusedContainerColor = Color.White,
+                    focusedIndicatorColor =  Color.Transparent,
+                    unfocusedIndicatorColor = Color.Transparent,
+                    disabledIndicatorColor = Color.Transparent),
+                shape = RoundedCornerShape(15.dp),
+                placeholder = {
+                    Text("Please enter username", color = blue)
+                },
+                modifier = Modifier
+                    .height(50.dp)
+                    .width(250.dp)
+                    .constrainAs(username) {
+                        top.linkTo(babbleTitle.bottom, margin = (maxHeight/10).dp)
+                        absoluteLeft.linkTo(
+                            parent.absoluteLeft,
+                            margin = (maxWidth/2 - 250/2).dp)
+                    })
+
+            // user desc.
+            TextField(
+                value = passwordText,
+                onValueChange = { passwordText = it },
+                singleLine = true,
+                colors = TextFieldDefaults.colors(
+                    Color.Black,
+                    focusedContainerColor = Color.White,
+                    unfocusedContainerColor = Color.White,
+                    focusedIndicatorColor =  Color.Transparent,
+                    unfocusedIndicatorColor = Color.Transparent,
+                    disabledIndicatorColor = Color.Transparent),
+                shape = RoundedCornerShape(15.dp),
+                placeholder = {
+                    Text("Please enter password", color = blue)
+                },
+                visualTransformation = PasswordVisualTransformation(),
+                modifier = Modifier
+                    .height(50.dp)
+                    .width(250.dp)
+                    .constrainAs(password) {
+                        top.linkTo(username.bottom, margin = (maxHeight/20).dp)
+                        absoluteLeft.linkTo(
+                            parent.absoluteLeft,
+                            margin = (maxWidth/2 - 250/2).dp)
+                    })
+
+            // Button for viewing my followers
```

Comment:
typo in comment from maybe a copy/paste?

Reply:
Yes! I can remove that.